### DISCUSSION TITLE
What app or editor can help in scaling font families digitally, the whole glyph set at once?

### DISCUSSION THREAD

**Discussion by cuneiform100** [fontforge/fontforge#5448] [Q&A]
Hi, Everyone

The task is scaling a font down to a font family just while reducing the glyph sizes  **_h o r i z o n t a l l y_** , thus getting each time fonts reduced to - say - 25%, 50%, 75% out of their normal sizes.

Is it possible to do so  in FontForge or any other app known so far **_through global setting options_** in order to escape any separate working on each glyph?

Any ideas? What about AI? 

<img width="255" alt="Potsdm A-Z Full Stepping 2x_red" src="https://github.com/user-attachments/assets/343e75d9-f260-4c91-b3bb-1a305abe6782">

**Comment by 2-www** [comment]
- **fontforge**
just select all glyphs (ctrl+a), open transform (ctrl+\\), select "scale" in the dropdown, and write your scale in "X %" and leave "Y %" 100%

>What about AI?

"ai"? if you mean neural networks, they would probably very much mess up the font in the process
